tox.ini

```diff
@@ -126,7 +126,7 @@ commands =
     functests: sh bin/create-testdb h_functests
     {tests,functests}: python3 -m h.scripts.init_db --delete --create
     tests: python -m pytest --cov --cov-report= --cov-fail-under=0 --numprocesses logical --dist loadgroup {posargs:tests/unit/}
-    functests: pytest {posargs:tests/functional/}
+    functests: python -m pytest {posargs:tests/functional/}
```

Comment:
This was necessary to fix an h-testkit import error. This is the more correct form of the command anyway (and what the cookiecutter does)